### DISCUSSION TITLE
How to add auth to apps created before `0.18.0`?

### DISCUSSION THREAD

**Discussion by flybayer** [blitz-js/blitz#1090] [Q&A] [answered]
If I started my Blitz app before `0.18.0`, I didn't have auth set up by default. How can I add auth to my app now?

**Comment by flybayer** [comment, marked as answer]
Hey, great question Brandon! 

There's two main ways:
1. See diff
   - `npm i -g blitz@latest`
   - `blitz new your-existing-app-name`
   - Any totally new files will be created. And existing files with changes will show you a diff and ask whether you want to overwrite or not. 
2. Generate and copy
   - `npm i -g blitz@latest`
   - `blitz new newApp`
   - Compare files between your existing app and this newApp. And manually copy over files from the new app to your app. Pay attention to differences in installed dependencies and blitz.config.js.

**Reply by thomasbroderick** [reply]
This would make a great addition to the official Docs under "Upgrading from earlier versions" or similar heading.  I ended up doing something similar upgrading a few weeks back, but it'd be nice to have some official guidance.
